src/BuildScriptGenerator/PhpVersions.cs

```diff
@@ -17,7 +17,7 @@ public static class PhpVersions
         public const string Php81TarSha256 = "a1317eff0723a2b3d3122bbfe107a1158570ea2822dc35a5fb360086db0f6bbc";
         public const string Php80Version = "8.0.17";
         public const string Php80Keys = "1729F83938DA44E27BA0F4D3DBDB397470D12172 BFDDD28642824F8118EF77909B67A5C12229118F";
-        public const string Php80TarSha256 = "bdbd792901c156c4d1710c9d266732d3c17f6ff63850d6660b9d8d3411188424";
+        public const string Php80TarSha256 = "f27a2f25259e8c51e42dfd74e24a546ee521438ad7d9f6c6e794aa91f38bab0a";
```

Comment:
I'm a bit confused about the whole 8.0.17 vs. 8.0.16 thing. If we're already supporting 8.0.17, could you please clarify why we should support 8.0.16?
Also, could you please clarify why we're updating the SHA here if we're keeping the current PHP version as 8.0.17?

Reply:
I confirmed with the team `8.0.17` is fine. Closing this PR